nmtwizard/preprocess/prepoperator.py

```diff
@@ -110,9 +114,11 @@ def apply(self, tu_batch):
 
 class Filter(TUOperator):
 
-    def __init__(self):
+    def __init__(self, process_type):
         # TODO: Sub-criteria for source_detok, target_detok, source_tok, target_tok, or both with alignment ?
         self._criteria = []
+        if process_type=="inference" or process_type=="postprocess":
```

Comment:
Avoid passing the process_type as constructor argument if possible. A base method that can be overridden such as:

```python
def is_enabled_for(self, process_type):
  return True
```

seems a cleaner to implement this.